### DISCUSSION TITLE
Visual feedback when using Type outside of "inserts" and Keys

### DISCUSSION THREAD

**Discussion by nocksock** [charmbracelet/vhs#72] [Ideas]
When creating tapes for eg. nvim plugins, cli applications with a tui etc, it would be neat to have som way to show the keys being pressed. 

Something along those lines:

```
Set ShowKeys 0
Type "nvim README.md"
Set ShowKeys 1
Enter
Type "/something"
Enter
Type "nn"
Type "ciw"
Set ShowKeys 0
Type "foobar"
Set ShowKeys 1
Escape
```

Maybe some kind of Overlay that can be placed freely that also groups keys by lines: eg `nn` would be shown, and then replaced by `ciw` - instead of showing `nnciw` - then vanish until `Escape`, which is shown too. Or when navigating inside glow, or btop etc.

Theres probably a much more elegant way to write this syntax-wise. :D

**Comment by maaslalani** [comment]
This is a really great idea! It would be quite useful for tutorials and other types of demos! Thank you for suggesting it!

**Comment by BlackEdder** [comment]
This would be very useful, has it ever been added?
